src/core/TrainController.hpp

```diff
@@ -71,19 +69,19 @@ namespace godot {
 
             Dictionary get_config() const;
             void update_config(const Dictionary &p_config);
-            void set_config_property(String &key, Variant &p_value);
+            void set_config_property(String &key, Variant &p_value) {};
```

Comment:
This should not be implemented as empty method. This is unused method, so declaration can be safely removed.